### PR TITLE
fix(plugin-template): new plugins fail the verify-docs & verify-types check

### PR DESCRIPTION
**Pull request by aramissennyeydd**
## Hey, I just made a Pull Request!

There is a block in the new plugin readme's that is causing issues for the verify-docs check and doesn't redirect to the desired relative `/dev` folder.

I also moved the `@types/express` dependency to a devDependency.

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: .changeset/shy-hounds-battle.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/cli': patch
```
Comment (Rickyciber) [inline .changeset/shy-hounds-battle.md:2]: hey

**Comment by aramissennyeydd** [comment]
@benjdlambert Any chance this can go out in the next release 🙏 ? Small change to make it easier to create plugins that sit in this repo.

**Comment by aramissennyeydd** [comment]
bump

**Review by benjdlambert — APPROVED** [review]
Nice thanks @aramissennyeydd! :tada:

**Pull request merged**